vidarr-niassa/src/main/java/ca/on/oicr/gsi/vidarr/niassa/NiassaOutputProvisioner.java

```diff
@@ -42,6 +43,8 @@
           Map.entry("txt/junction", "text/junction"),
           Map.entry("txt/plain", "text/plain"));
 
+  private static Map<String, Semaphore> mkdirSemaphores = Map.of();
```

Comment:
`Map.of` is unmodifiable and not concurrent-safe anyway. Just create a single semaphore.

Reply:
Fixed